### DISCUSSION TITLE
How is presale_start and public_sale_start duration calculated ?

### DISCUSSION THREAD

**Discussion by akshay-anv** [TENK-DAO/tenk#51] [Q&A]
Hi Loving the new updated contract with all the features but one thing I am not able to figure out is how is presale_start and public_sale_start duration calculated ?  Is 86400 equal to 1 day ? Please elaborate more 🙏 . Thanks for helping the near community out with this amazing repo

**Comment by willemneal** [comment]
There is a PR to improve the types for the start variables.  They are unix timestamps in ms.  See here: https://github.com/TENK-DAO/tenk/blob/89b473e56a1c245b2296e832474d9947e3574dbb/scripts/deploy.ts#L18
